sklearn/feature_extraction/tests/test_text.py

```diff
@@ -892,6 +892,17 @@ def test_countvectorizer_vocab_dicts_when_pickling():
         assert_equal(cv.get_feature_names(), unpickled_cv.get_feature_names())
 
 
+def test_countvectorizer_string_object_as_input():
+    # Ensure that string object is not given as input.
+    cv = CountVectorizer()
```

Comment:
how about ```
for vec in [CountVectorizer(), TfidfVectorizer(), HashingVectorizer()]:
```
?

(Some unit testing conventions don't like such looks, but we're fine with them here...)